python-backend/tests/document_manager/resources/test_document_manager_resource.py

```diff
@@ -74,12 +80,57 @@ def test_bad_file_type(test_client, auth_headers, setup_info):
     }
 
     post_resp = test_client.post('/document-manager', data=data, headers=auth_headers['full_auth_header'])
+    
+    assert post_resp.status_code == 200
+
     post_data = json.loads(post_resp.data.decode())
     filename = post_data['document_manager_guids'][0]+'.docx'
-    saved_file = open(f'{test_dir_path}/mines/{setup_info.get("mine_guid")}/tailings/{filename}')  
+    saved_file = open(f'{test_dir_path}/mines/{setup_info.get("mine_guid")}/tailings/{filename}')
     
     assert len(post_data['errors']) == 2
     assert len(post_data['document_manager_guids']) == 1
     assert saved_file
+
+def test_download_file(test_client, auth_headers, setup_info):
+    test_dir = os.mkdir(setup_info.get('base_path') + '/test')
+    test_dir_path = setup_info.get('base_path') + '/test'
     
-    shutil.rmtree(test_dir_path)
+    data = {
+        'folder': f'test/mines/{setup_info.get("mine_guid")}/tailings',
+        'file': setup_info.get('file_upload_1'),
+        'pretty_folder': f'test/mines/{setup_info.get("mine_no")}/tailings',
+    }
+    
+    post_resp = test_client.post('/document-manager', data=data, headers=auth_headers['full_auth_header'])
+    
+    assert post_resp.status_code == 200
+
+    post_data = json.loads(post_resp.data.decode())
+
+    assert len(post_data['document_manager_guids']) == 1
+    
+    filename = post_data['document_manager_guids'][0]+'.pdf'
+    saved_file = open(f'{test_dir_path}/mines/{setup_info.get("mine_guid")}/tailings/{filename}', 'rb') 
+
+    get_resp = test_client.get(f'/document-manager/{post_data["document_manager_guids"][0]}', headers=auth_headers['full_auth_header'])
+    saved_file.seek(0)
+
+    assert get_resp.data == saved_file.read()
+
+def test_download_file_no_guid(test_client, auth_headers, setup_info):
+    
+    get_resp = test_client.get(f'/document-manager', headers=auth_headers['full_auth_header'])
+    get_data = json.loads(get_resp.data.decode())
+
+    assert get_resp.status_code ==200
```

Comment:
If you have to push a change for the cleanup, just add a space to this assertion and the same one in the test below.

Reply:
fixed